test/dynamo/test_misc.py

```diff
@@ -5795,7 +5795,9 @@ def g2(a, b):
             return a + b + c
 
         def count_graph_break_msgs(msgs):
-            return sum(msg.find("Graph break") != -1 for msg in msgs)
+            return sum(
+                "Graph break" in msg and "details suppressed" not in msg for msg in msgs
```

Comment:
?

Reply:
Filtering out details suppressed graph break messages (to comply with removing duplicate detailed graph break messages)

Reply:
Simplified this check.